README.md

```diff
@@ -172,6 +172,7 @@ The `AudioIsolate` instance has the duty of receiving commands and sending them
 | **setVolume**| ({PlayerErrors error, double volume})| `int` handle, `double` volume| set  [handle] volume.|
 | **getIsValidVoiceHandle**| ({PlayerErrors error, bool isValid})| `int` handle| Check if a handle is still valid.|
 | **setVisualizationEnabled**| -| `bool` enabled| Enable or disable getting data from `getFft`, `getWave`, `getAudioTexture*`.|
+| **getVisualizationEnabled**| ({PlayerErrors error, bool isEnabled})| -| Get the state of visualization flag.|
```

Comment:
(This reminds me I haven't updated the README with the new changes. I'll do it later, I promise.